front_end/audits2/Audits2Panel.js

```diff
@@ -1,43 +1,242 @@
 // Copyright (c) 2016 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+
+/**
+ * @typedef {{
+ *     lighthouseVersion: ?string,
+ *     generatedTime: !string,
+ *     initialUrl: !string,
+ *     url: !string,
+ *     audits: ?Object,
+ *     aggregations: !Array.<*>
```

Comment:
another optional nit: closure and TS recognize the shorter `!Array<*>` (dropping the `.`) too. That might be devtools style, too, because closure a few years ago required the `.`